README.adoc

```diff
@@ -150,6 +150,7 @@ metadata fields:
 |===
 |`name` | Optional. Use this field to name of the metric.
 |`description` | Optional. Use this field to describe the purpose of the metric.
+|`absolute` | Optional. Use this field to determine whether the metric name is the exact name specified in the `name` field or with the package prefix.
```

Comment:
How about..."Use this field to determine whether the metric name is the exact name **that is** specified in the `name` field or **that is specified** with the package prefix."

Reply:
updated